torchvision/models/alexnet_onlyFullyConnected.py

```diff
@@ -0,0 +1,48 @@
+import torch
+import torch.nn as nn
+from .utils import load_state_dict_from_url
+
+
+__all__ = ['AlexNet', 'alexnet']
+
+
+model_urls = {
+    'alexnet': 'https://download.pytorch.org/models/alexnet-owt-4df8aa71.pth',
+}
+
+
+class AlexNet(nn.Module):
+
+    def __init__(self, num_classes=1000):
+        super(AlexNet, self).__init__()
+        
+        self.classifier = nn.Sequential(
```

Comment:
Why would you have the `classifier` as submodule if your whole module is nothing more than a classifier?